benchmark_runner/common/clouds/BM/bare_metal_exceptions.py

```diff
@@ -0,0 +1,32 @@
+
+class BMError(Exception):
+    """ Base class for all Bare metal error classes.
+        All exceptions raised by the benchmark runner library should inherit from this class. """
+    pass
+
+
+class MissingMasterNodes(BMError):
+    """
+    This class exception is for missing master nodes
+    """
+    def __init__(self):
+        self.message = 'There are missing master nodes'
```

Comment:
If you can identify which nodes are missing, enumerate them here.

Reply:
Its not so important which master node is missing, cluster is corrupted and the user should investigate it by himself

Reply:
Please list which node(s) are missing.  It could be a transitory or infrequent error, where having that information may help in debugging.

Reply:
This code is working properly more than 1 year, just move it to bare metal class.
I dont see any impact to return the missing name, It will require to hold the expected master/ worker nodes.

Reply:
It always helps to make it easier to debug.

Reply:
Fixed